### DISCUSSION TITLE
Is Uno .NET7 ready when .NET8 is already coming soon?

### DISCUSSION THREAD

**Discussion by jbtdevgit** [unoplatform/uno#13159] [Q&A] [answered]
Hi,

I am currently upgrading my project from .NET6 to NET7, and I am already having troubles with the previous packages I am using.
It seems like the toolkits are still on `pre-release` and when I create a new project using the template it  installs the NET6 packages image shown below. As of right now .NET6 is already out of support and .NET8 is already coming in November, how come we still have .NET6 packages for .NET7 project template?
![image](https://github.com/unoplatform/uno/assets/44351076/85fb7012-45ae-4724-adac-a6fad9576817)

Even the `InitializeComponent()` is having troubles to build with Android and Mac Catalyst, any workaround for this?
![image](https://github.com/unoplatform/uno/assets/44351076/9f1ce75e-9aff-4691-98b0-6efebce0ea04)

Below are the .NET 7 toolkit
https://www.nuget.org/packages/Uno.CommunityToolkit.WinUI/7.1.200-dev.6.gf657bb2e91
https://www.nuget.org/packages/Uno.Toolkit.UI/4.0.0-dev.64

**Comment by limeniye** [comment]
Good day, @jbtdevgit. 
Try to do the following steps:
https://github.com/unoplatform/uno/discussions/13152#discussioncomment-6659581

**Reply by jbtdevgit** [reply]
Thanks for this @limeniye , been scraping my head upgrading packages.. I'll take a look at it.

**Reply by jbtdevgit** [reply]
Hi @limeniye , `Community.Toolkit.WinUI` for windows is still in.NET5 https://www.nuget.org/packages/CommunityToolkit.WinUI/7.1.2 . Is this fine to use?

**Reply by jbtdevgit** [reply, marked as answer]
Nope, none of the packages are working, even with bare minimum project. Opened an issue https://github.com/unoplatform/uno/issues/13175

**Reply by limeniye** [reply]
> Hi @limeniye , `Community.Toolkit.WinUI` for windows is still in.NET5 https://www.nuget.org/packages/CommunityToolkit.WinUI/7.1.2 . Is this fine to use?


Currently, I'am using the following versions:
```XML
<CommunityToolkitVersion>7.1.2</CommunityToolkitVersion>

<UnoCommunityToolkitVersion>7.1.200-dev.6.gf657bb2e91</UnoCommunityToolkitVersion>
```

Yes, it's ok. The version of your target framework should be higher than .net-4.0, if you want to use this package.

**Comment by Charlotte-br560** [comment]
It's understandable to encounter frustrations when transitioning between .NET versions. Uno Platform's compatibility with .NET7 may vary, especially considering the rapid evolution of the .NET ecosystem. Regarding the persistence of .NET6 packages in the .NET7 project template, it's likely due to various factors such as compatibility concerns or development priorities.
